dev/DynamicDependency/MddCore.Architecture.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !defined(MDDCORE_ARCHITECTURE_H)
+#define MDDCORE_ARCHITECTURE_H
+
+namespace MddCore
+{
+enum class Architecture : int32_t // per PACKAGE_ID.processorArchitecture
```

Comment:
Why do we need another enum here if Windows::System::ProcessorArchitecture is what we're aliasing?

Reply:
Windows::System::ProcessorArchitecture is a list of values e.g. Arm64=12, X64=9, X86=0
I need a list of potentially multiple values - thus a bitfield or an array. Bitfield made more sense.